set.mm

```diff
@@ -72708,18 +72708,18 @@ Power Set ( ~ ax-pow ).  (Contributed by Mario Carneiro, 20-May-2013.)
     $( Existence of an operation class abstraction (special case).
        (Contributed by FL, 17-May-2010.)  (Revised by Mario Carneiro,
        1-Sep-2015.) $)
-    mpt2exg $p |- ( ( A e. R /\ B e. S ) -> F e. _V ) $=
-      ( wcel cvv wral elex ralrimivw syl mpt2exxg sylan2 ) DGJZCFJDKJZACLZHKJRS
-      TDGMSSACDKMNOABCDEFKHIPQ $.
+    mpoexg $p |- ( ( A e. R /\ B e. S ) -> F e. _V ) $=
+      ( wcel cvv wral elex ralrimivw syl mpoexxg sylan2 ) DGJZCFJDKJZACLZHKJRST
+      DGMSSACDKMNOABCDEFKHIPQ $.
   $}
 
   ${
     $d x y A $.  $d x y B $.
     $( If the domain of a function given by maps-to notation is a set, the
        function is a set.  (Contributed by NM, 12-Sep-2011.) $)
```

Comment:
see remark for corresponding theorems in iset.mm

Reply:
updated, including mpoexw